packages/paste-core/components/avatar/src/index.tsx

```diff
@@ -152,3 +155,4 @@ Avatar.propTypes = {
 };
 
 export {Avatar};
+export {AvatarGroup} from './AvatarGroup';
```

Comment:
suggestion: Let's rename this file to `Avatar.tsx` and add a new `index.tsx` file with these exports. It's the new pattern we've been mostly following and makes this file more clear.